supabase/migrations/06_connectors.sql

```diff
@@ -1,14 +1,27 @@
+create domain jsonb_internationalized_value as jsonb check (
+  (value is null) OR -- This feels wrong, but without it the check constraint fails on nulls
```

Comment:
A suggestion:

What about adding the new columns with `generated always as (json_build_object('en-US',open_graph->'en-US'->>'title')) stored` constraints? In this first migration they would co-exist with and depend on `open_graph` and this guarantees they stay in sync and back-filled from our existing data updates.

That also lets you make `title`, `logo_url` et al `non null`, because the expression can resolve to an empty string if `open_graph` itself is null,
and you can remove this kludge.

Then, in the migration that drops `open_graph`, you drop the generating expression from these columns.

[Here's a thread with further details on how that can work.](https://dba.stackexchange.com/questions/294850/convert-a-postgres-generated-column-to-a-regular-non-generated-column-in-place)

Reply:
Done!